src/System.Diagnostics.EventLog/src/System/Diagnostics/NativeMethods.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Win32
+{
+    internal static class NativeMethods
+    {
```

Comment:
It would be great if we could get rid of this class completely as in Desktop it kept all the PInvokes. I suggest 1) checking if all consts are being used and remove the unused ones and 2) renaming the class.

Reply:
And cleaning the empty lines at the end.